src/c.zig

```diff
@@ -1,4 +1,4 @@
-const std = @import("std");
+const std = @import("std")
```

Comment:
Is this intentional? :thinking:

Reply:
yes

Reply:
i'm trying to test the bot works

its supposed to comment on the PR saying we need to run zig fmt and also prettier